### DISCUSSION TITLE
Set the opacity value for DragOverlay

### DISCUSSION THREAD

**Discussion by Adam-Collier** [clauderic/dnd-kit#1024] [Q&A]
Hi guys,

Quick question here and I hope you can help! How can I set the opacity value for the DragOverlay? I've tried setting it via the styles and className props but to no avail

```jsx
<DragOverlay dropAnimation={null} style={{ opacity: 1 }}>
  {isActiveElement ? (
    <Element />
  ) : null}
</DragOverlay>
```

**Comment by eessadrati** [comment]
@Adam-Collier  same here, did you find any solution??

**Comment by samirergaibi** [comment]
Setting the background worked for me

**Comment by danielstocks** [comment]
I know this is an old question, but in case anyone lands here from search, what I ended up doing:

```jsx
<DragOverlay
  dropAnimation={{
    ...defaultDropAnimation,
    sideEffects: defaultDropAnimationSideEffects({
      styles: {
        active: {
          opacity: "1",
        },
      },
    }),
  }}
>
  {isActiveElement ? (
    <Element />
  ) : null}
</DragOverlay>
```

The modules `defaultDropAnimation` and `defaultDropAnimationSideEffects` can be imported from `@dnd-kit/core`

**Reply by JamesPardue** [reply]
This resolved the problem I was having as well. Thanks!
